hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -77,3 +77,27 @@ check_cov -waiver -add -source_file {../src/lowrisc_prim_diff_decode_0/rtl/prim_
 # parasitic state. If that is no longer the case, this assertion will fail.
 assert -name PrimDiffDecodeNoParasiticState_A\
  {dut.gen_alert_tx[0].u_prim_alert_sender.u_decode_ping.gen_async.state_q < 3}
+
+# Task for onehot0Check_A and EnableCheck_A assertions.
+task -create notOnehotInpt -copy {
+  .*\.u_prim_reg_we_check.u_prim_onehot_check.Onehot0Check_A
+  .*\.u_prim_reg_we_check.u_prim_onehot_check.Onehot0Check_A:precondition1
+  .*\.u_prim_reg_we_check.u_prim_onehot_check\..*\.EnableCheck_A
+  .*\.u_prim_reg_we_check.u_prim_onehot_check\..*\.EnableCheck_A:precondition1
+} -regexp
+
+# This will let the preconditions for Onehot0check_A and EnableCheck_A happen.
+stopat -task notOnehotInpt dut.u_reg.reg_we_check
+
+# Disabling the affected assertions and related covers from the embedded task.
+cover -disable -regexp\
```

Comment:
For an easier way to do this, I'd suggest you copy the `move_to_task` function from `hw/ip/rom_ctrl/dv/formal/verify.tcl`, which does the copying and disabling together.

Reply:
Sorry, there is a way to waive an instance after initialization
```check_cov -waiver -add -instance {instance} -comment {comment}```

Problem is that the  _**-instance**_  doesn't allow regexp. Based on _NumAlerts_ parameter, we can have more instances of _prim_alert_sender_. We can do that through tcl loop iterating through _NumAlerts_ and generating _**check_cov -waiver**_ command per iteration.